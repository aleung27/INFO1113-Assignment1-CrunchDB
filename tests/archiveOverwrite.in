SET f 4
SET 6 6
ARCHIVE 1 archiveOverwrite.txt
SNAPSHOT
ARCHIVE 0 archiveOverwrite.txt
ARCHIVE 1 archiveOverwrite.txt
BYE
