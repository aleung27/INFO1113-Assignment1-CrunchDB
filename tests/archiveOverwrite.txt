f|4
6|6
