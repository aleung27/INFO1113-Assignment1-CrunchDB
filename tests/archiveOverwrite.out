> ok

> ok

> no such snapshot

> saved as snapshot 1

> no such snapshot

> ok

> bye
